### PR TITLE
fix: change collection to collection.id

### PR DESCRIPTION
**Pull request by adong**
## Context

Fix a small bug that adding pipeline to collection.

## Objective
This PR fixed the function arguments from model to `model.id`.

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: app/pipeline/controller.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ export default Controller.extend({
   pipeline: alias('model.pipeline'),
   collections: alias('model.collections'),
   actions: {
-    addToCollection(pipelineId, collectionId) {
-      return this.store.findRecord('collection', collectionId).then(collection => {
+    addToCollection(pipelineId, c) {
+      return this.store.findRecord('collection', c.id).then(collection => {
```
Comment (jithin1987) [inline app/pipeline/controller.js]: you have the collection here. Then why fetch it again ?

Comment (adong) [reply app/pipeline/controller.js]: I thought it was for fetching new collection's data. Just removed it.

@Code-Beast 🤔

Comment (Code-Beast) [reply app/pipeline/controller.js]: it's the collection the pipeline is added to

File: app/pipeline/controller.js [hunk 2]
```diff
@@ -8,16 +8,10 @@ export default Controller.extend({
   pipeline: alias('model.pipeline'),
   collections: alias('model.collections'),
   actions: {
-    addToCollection(pipelineId, collectionId) {
-      return this.store.findRecord('collection', collectionId).then(collection => {
-        const pipelineIds = collection.get('pipelineIds');
+    addToCollection(pipelineId, collection) {
+      collection.pipelineIds.push(pipelineId);
 
-        if (!pipelineIds.includes(pipelineId)) {
```
Comment (jithin1987) [inline app/pipeline/controller.js]: we should keep this. check.

File: app/pipeline/controller.js (hunk 1 above)
Comment (adong) [reply app/pipeline/controller.js]: If it only added to the current user's collection, then maybe we dont need to fetch.

Comment (Code-Beast) [reply app/pipeline/controller.js]: @adong Oh, just notice it is addToCollection in pipeline/controller.js, I thought this was the addMultipleToCollection method in dashboard/show

**Pull request merged**